packages/web-app-files/src/components/AppBar/ViewOptions.vue

```diff
@@ -18,7 +18,7 @@
       :aria-label="toggleSidebarButtonLabel"
       variation="passive"
       appearance="raw"
-      class="oc-ml-s oc-py-s oc-mb-xs"
+      class="oc-my-s oc-mb-xs oc-px-xs pc-py-xs"
```

Comment:
```suggestion
      class="oc-my-s oc-mb-xs oc-px-xs oc-py-xs"
```

Spotted a typo, also I think these can be simplified and don't look as expected (very little padding)